Makefile

```diff
@@ -372,11 +367,20 @@ endif
 
 ifdef PUBLISH_AWS_S3
 publish-aws-s3: $(PUBLISH_AWS_S3)
+ifeq ($(RELEASE_TYPE),production)
 	$(foreach publishable,$^,$(call execute-command,./scripts/publish/aws-s3.sh \
 		-f $(publishable) \
 		-b $(S3_BUCKET) \
 		-v $(APPLICATION_VERSION) \
 		-p $(PRODUCT_NAME)))
+else
```

Comment:
Should this explicitly be `ifeq ($(RELEASE_TYPE),snapshot)` rather than `else` ?  (in case we decide to add other RELEASE_TYPEs in future)